src/libraries/System.Drawing.Common/tests/AssemblyInfo.cs

```diff
@@ -5,4 +5,4 @@
 using Xunit;
 
 [assembly: ActiveIssue("https://github.com/dotnet/runtime/issues/35917", typeof(PlatformDetection), nameof(PlatformDetection.IsMonoInterpreter))]
-[assembly: SkipOnMono("System.Drawing.Common is not supported on Browser", TestPlatforms.Browser)]
+[assembly: SkipOnPlatform(TestPlatforms.Browser, "System.Drawing.Common is not supported on Browser")]
```

Comment:
Can we change these assembly-wide exclusions to go through the `<ProjectExclusions>` items in `tests.proj`?

That way we don't go through the whole process of packaging them up and sending them to Helix just to discover that we don't need to run any of them.

Reply:
That's what the `IgnoreForCI` csproj property that I discussed with @safern does :)